src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -206,6 +208,7 @@ debug_configuration::debug_configuration()
     get_gpu_debug_env_var("DumpIteration", dump_iteration_str);
     std::string mem_preallocation_params_str;
     get_gpu_debug_env_var("MemPreallocationOptions", mem_preallocation_params_str);
+    get_gpu_debug_env_var("DisableWinogradConv", disable_winograd_convolution);
```

Comment:
nit: variable name and command is slightly different. What about using same name (`conv` or `convolution`) for both?

Reply:
Applied